docs/icicle/overview.md

```diff
@@ -6,45 +6,42 @@ ICICLE allows developers with minimal GPU experience to effortlessly accelerate
 
 ICICLE has been used by many leading ZK companies such as [Celer Network](https://github.com/celer-network), [Gnark](https://github.com/Consensys/gnark), [EZKL](https://blog.ezkl.xyz/post/acceleration/) and others to accelerate their ZK proving pipeline.
 
-Dont have a GPU? [read this](#dont-have-access-to-a-gpu)
 
-## How can you use ICICLE?
+## Dont have access to a GPU?
+
+We understand that not all developers have access to a GPU and we don't want this to limit anyone from developing with ICICLE. 
+Here are some ways we can help you gain access to GPUs: 
+### Grants
+
+At Ingonyama we are interested in accelerating the progress of ZK and cryptography. If are an engineer, developer or an academic researcher we invite you to checkout [our grant program](https://www.ingonyama.com/blog/icicle-for-researchers-grants-challenges). We will give you access to GPUs and even pay you to do your dream research!
 
-ICICLE can be used in the same way you would use any other cryptography library. However, over the past couple of months developing and integrating ICICLE into many systems we found a couple use case catagories.
+### Google Colab
 
-### ICICLE for circuit developers
+This is a great way to get started with ICICLE instantly. Google Colab offers free GPU access the down side being that the GPU doesn't have a lot of RAM but it should be enough for experimenting and even prototyping with ICICLE.
+
+For an extensive guide on how to setup Google Colab with ICICLE refer to [this article](https://github.com/gkigiermo/rust-cuda-colab).
 
+
+_If none of these options suit your needs, contact us on [telegram](https://t.me/RealElan) for assistance. We're committed to ensuring that a lack of a GPU doesn't become a bottleneck for you. If you need help with setup or any other issues, we're here to do our best to help you._
```

Comment:
it might be better to use an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) than just italics for a note/tip